src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/model/response/ScheduleItem.kt

```diff
@@ -0,0 +1,100 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.response
+
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request.AppointmentType
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request.BookingType
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request.CourtHearingType
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request.ProbationMeetingType
+import java.time.LocalDate
+import java.time.LocalTime
+
+@Schema(description = "An item on a prison schedule (all prison appointments and their booking details)")
+data class ScheduleItem(
+  @Schema(description = "The internal ID for the video booking", example = "123")
+  val videoBookingId: Long,
+
+  @Schema(description = "The internal ID for an appointment related to a booking. It is unique in this list", example = "123")
+  val prisonAppointmentId: Long,
+
+  @Schema(description = "The booking type", allowableValues = ["COURT", "PROBATION"], example = "COURT")
```

Comment:
You don't need allowable values on enums.  Can remove off booking type and booking status.

Reply:
Done.